apps/examples/iotivity_simpleserver/README.md

```diff
@@ -0,0 +1,49 @@
+# IoTivity Sample apps - simple server
+
+This is an example for an IoTivity simple server test.
+
+## How to build
+Configure the build from $TIZENRT_BASEDIR/os/tools directory.
+```bash
+./configure.sh artik053/iotivity
+```
+
+Select the 'Enable build IoTivity Samples' example in menuconfig.
+
+    -+-Application Configuration
+          |
+          +-Examples
+               |
+               +-Enable build IoTivity Samples
+
+
+## How to test
+### Server (Artik053)
```

Comment:
How about change Artik053 to thing or Tizen RT? It looks it is working at only ARTIK053.